libs/core/src/main/java/org/opensearch/core/index/Index.java

```diff
@@ -59,6 +59,7 @@ public class Index implements Writeable, ToXContentObject {
     public static final Index[] EMPTY_ARRAY = new Index[0];
     private static final String INDEX_UUID_KEY = "index_uuid";
     private static final String INDEX_NAME_KEY = "index_name";
+    public static final String UNKNOWN_INDEX_NAME = "unknown";
```

Comment:
What happens if an index is created with name `unknown`?

Reply:
Possible, let me rename it to `_unknown_` as we don't allow it to start with underscore.